pkgs/applications/networking/instant-messengers/discord/default.nix

```diff
@@ -13,37 +13,38 @@ let
   srcs = rec {
     x86_64-linux = {
       stable = fetchurl {
-        url = "https://dl.discordapp.net/apps/linux/${version}/discord-${version}.tar.gz";
+        url =
```

Comment:
These break lines seem unnecessary to me and also make diff bigger. Otherwise LGTM

Reply:
Yes, you're right, it blows up the commit. It was auto formatted by `nixfmt`. Reverted these lines.
Are there any requirements to a default formatting?

Reply:
nixpkgs-fmt is the most used one afaik

There's a rfc which will codify a formatter https://github.com/NixOS/rfcs/pull/101